### ISSUE TITLE
Error: Unsupported type registry preset

### ISSUE THREAD

**Issue by iicc1** [stakelink/substrate-payctl#4]
Hello,

I have used this tool with Kusama and works fine, but I have tried to use it with Stafi, a Substrate based blockchain, and payctl throws this error when I list or claim rewards:

```
payctl -c stafi_payouts.conf list
...
   raise ValueError(f'Unsupported type registry preset "{name}"')
ValueError: Unsupported type registry preset "stafi"
```

This is my config file:
```
[Defaults]
RPCURL = wss://mainnet-rpc.stafi.io/
Network = stafi
...
```

Maybe the issue is that [Stafi uses custom types](https://github.com/stafiprotocol/stafi-bootstrap#types)? In that case, the tool config would need to specify a custom type registry JSON.

Many thanks for the great tool!

**Comment by apuigsech** [comment]
I have tested payctl on Kusama an Polkadot. I thought it should be compatible with any Substrate network with the Staking module but it seems it may not be.

I will investigate a bit on this issue, but it's on my lowest priority.

**Comment by apuigsech** [comment]
Can you try again using the last substrateinterface version?

**Comment by iicc1** [comment]
> 
> 
> Can you try again using the last substrateinterface version?

How can I update substrateinterface? Does it get updated if I reinstall Payctl?

**Comment by apuigsech** [comment]
Fixed in #12.
